examples/snippets/modules/ROOT/examples/sensor-data-scala/src/main/scala/sensordata/ConfigParams.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sensordata
+
+import cloudflow.streamlets._
+
+trait ConfigParams {
```

Comment:
Its not clear whose configparams these are.. I would prefer to define these where they are used.

Reply:
Oh it also looks like (unless there is a 'too big diff' in here) that it is not used?